web/docs/project/env-vars.md

```diff
@@ -6,14 +6,14 @@ title: Env Variables
 
 For instance, _during development_, you may want your project to connect to a local development database running on your machine, but _in production_, you may prefer it to connect to the production database. Similarly, in development, you may want to use a test Stripe account, while in production, your app should use a real Stripe account.
 
-While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes.
+While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes, and then access them in the code.
 
 In Wasp, you can use environment variables in both the client and the server code.
 ## Client Env Vars
 
-Client environment variables are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys).
+Client environment variables are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys -> you can provide those to server instead).
 
-To enable Wasp to pick them up, client environment variables must be prefixed with `REACT_APP_`, for example: `REACT_APP_SOME_VAR_NAME=...`.
+To enable Wasp to pick them up, client env vars must be prefixed with `REACT_APP_`, for example: `REACT_APP_SOME_VAR_NAME=...`.
```

Comment:
Why tho? :smile: 

You probably did it because you feel that `env vars` has become such a universal acronym that it's reached a point where it's more recognizable than the full name. I disagree, but fair enough.

I don't have a strong preference either way.

Still, we should then probably update all occurrences in the text to be consistent.

Reply:
Can't remember exactly, I think I felt "environemnt variables" was a mouthful here and this was in the part of docs where the naming "env vars" was already established, so kind of as you said.

We could update all the occurences, but they will again diverge at some point, since we don't have an automatic mechanism to check it or enforce it. So I wouldn't bother with it.